helpers.js

```diff
@@ -0,0 +1,61 @@
+const childProcess = require("child_process");
+const fs = require("fs");
+const pathmodule = require("path");
+const { v4: uuidv4 } = require("uuid");
+
+async function execTerraform(cmd, args, workDir) {
+  if (!workDir) {
+    throw new Error("Parameter Working Directory is required.");
```

Comment:
No, because if it DID, and you defined it in plugin settings and then left it empty in the action parameter, you'd get the error that the parameter is required. In a future platform release this should be fixed somehow. https://kaholo.atlassian.net/browse/KAH-3533